src/test/end-to-end/shared/testingSupportApi.ts

```diff
@@ -18,22 +17,87 @@ const createFile = (filePath, fileName) => {
+};
+
+export const clearAllPublicationsByTestPrefix = async (testSuitePrefix: string) => {
+    const tokenDataManagement = await getDataManagementCredentials();
+    try {
+        await superagent
+            .delete(`${testConfig.DATA_MANAGEMENT_BASE_URL}/testing-support/publication/${testSuitePrefix}`)
+            .set('Content-Type', 'application/json')
+            .set({ Authorization: 'Bearer ' + tokenDataManagement.access_token });
+    } catch (e) {
+        throw new Error(`Failed to delete publications test data , http-status: ${e.response?.status}`);
+    }
+};
+
+export const clearAllSubscriptionsByTestPrefix = async (testSuitePrefix: string) => {
+    const tokenSubscriptionManagement = await getSubscriptionManagementCredentials();
+    try {
+        await superagent
+            .delete(`${testConfig.SUBSCRIPTION_MANAGEMENT_BASE_URL}/testing-support/subscription/${testSuitePrefix}`)
+            .set('Content-Type', 'application/json')
+            .set({ Authorization: 'Bearer ' + tokenSubscriptionManagement.access_token });
+    } catch (e) {
+        throw new Error(`Failed to delete subscriptions test data , http-status: ${e.response?.status}`);
+    }
+};
+
+export const clearAllLocationsByTestPrefix = async (testSuitePrefix: string) => {
+    const tokenDataManagement = await getDataManagementCredentials();
+    try {
+        await superagent
+            .delete(`${testConfig.DATA_MANAGEMENT_BASE_URL}/testing-support/location/${testSuitePrefix}`)
+            .set('Content-Type', 'application/json')
+            .set({ Authorization: 'Bearer ' + tokenDataManagement.access_token });
+    } catch (e) {
+        throw new Error(`Failed to delete locations test data , http-status: ${e.response?.status}`);
+    }
+};
+
+export const clearAllAccountsByTestPrefix = async (testSuitePrefix: string) => {
+    const tokenDataManagement = await getAccountManagementCredentials();
+    try {
+        await superagent
+            .delete(`${testConfig.ACCOUNT_MANAGEMENT_BASE_URL}/testing-support/account/${testSuitePrefix}`)
+            .set('Content-Type', 'application/json')
+            .set({ Authorization: 'Bearer ' + tokenDataManagement.access_token });
+    } catch (e) {
+        throw new Error(`Failed to delete accounts test data , http-status: ${e.response?.status}`);
+    }
+};
+
+export const clearAllMediaApplicationsByTestPrefix = async (testSuitePrefix: string) => {
+    const tokenDataManagement = await getAccountManagementCredentials();
+    try {
+        await superagent
+            .delete(`${testConfig.ACCOUNT_MANAGEMENT_BASE_URL}/testing-support/application/${testSuitePrefix}`)
+            .set('Content-Type', 'application/json')
+            .set({ Authorization: 'Bearer ' + tokenDataManagement.access_token });
+    } catch (e) {
+        throw new Error(`Failed to delete applications test data , http-status: ${e.response?.status}`);
+    }
+};
+
 export const deleteLocation = async (locationId: string) => {
```

Comment:
Is this method still needed now we have the testing support one? Is that used in any other tests?

Reply:
I will remove this

Reply:
Removed deleteLocation, delete artefact and delete subscriptions methods